### PR TITLE
[Release] sdk/resourcemanager/workloadssapvirtualinstance/armworkloadssapvirtualinstance/0.1.0

### PR DESCRIPTION
**Pull request by Alancere**
https://github.com/Azure/sdk-release-request/issues/4843

### CODE REVIEW

**Comment by raych1** [comment]
The release request is held on.

**Review by tadelesh — APPROVED** [review]

**Comment by Alancere** [comment]
Hi @benbp, `/azp run go - armworkloadssapvirtualinstance - ci` does not work properly, how do I use it.

File: sdk/resourcemanager/workloadssapvirtualinstance/armworkloadssapvirtualinstance/ci.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+    - sdk/resourcemanager/workloadssapvirtualinstance/armworkloadssapvirtualinstance/
+
+stages:
```
Comment (benbp) [inline sdk/resourcemanager/workloadssapvirtualinstance/armworkloadssapvirtualinstance/ci.yml:24]:
```suggestion
extends:
```

Comment (benbp) [reply sdk/resourcemanager/workloadssapvirtualinstance/armworkloadssapvirtualinstance/ci.yml:24]: @Alancere this should fix your issue. We had to switch to extends recently to be compatible with newly required pipeline templates.

**Pull request merged**